everestrs/everestrs-build/jinja/module.jinja2

```diff
@@ -43,6 +43,12 @@ pub(crate) struct Module {
     publisher: ModulePublisher,
 }
 
+/// The context structure.
+pub struct Context<'a> {
+    pub publisher: &'a ModulePublisher,
+    pub name: &'a str,
```

Comment:
should this not be `implementation_id`. I am not so clear on EVerest nomenclatures.

Reply:
i'm also not sure